guides/v2.3/graphql/functional-testing.md

```diff
@@ -170,3 +170,126 @@ $registry->register('isSecureArea', false);
 
 A fixture rollback filename should correspond to the original fixture filename postfixed by `_rollback` keyword. (For example, `virtual_product_rollback.php`.) 
 Magento provides fixtures in the `dev/tests/integration/testsuite/Magento/<ModuleName>/_files` directory. Use these fixtures whenever possible. When you create your own fixture, also create a proper rollback.
+
+## Expected exceptions
+When it is necessary to cover negative test scenarios and ensure that testing elicits proper responses, define the expected exception message:
+  - in the body of test
+  - in test function annotation
+
+#### Define exception message expectation in the body of test
```

Comment:
Delete one of the #s. The previous heading was a H2, so this one should be a H3.

Reply:
**done**